src/stores/dashboardActions.ts

```diff
@@ -213,22 +225,24 @@ export const useBcrosDashboardActions = defineStore('bcros/dashboardActions', ()
           (
             isAllowedToFile(FilingTypes.RESTORATION, FilingSubTypeE.FULL_RESTORATION) ||
             isAllowedToFile(FilingTypes.RESTORATION, FilingSubTypeE.LIMITED_RESTORATION)
-          )
+          ) &&
+          isAuthorized(AuthorizedActionsE.RESTORATION_REINSTATEMENT_FILING)
         )
       }
 
       case AllowableActionE.STAFF_COMMENT: {
-        return (isBusiness && isStaffAccount)
+        return (isBusiness && isAuthorized(AuthorizedActionsE.STAFF_COMMENTS))
       }
 
       case AllowableActionE.TRANSITION: {
-        return isAllowedToFile(FilingTypes.TRANSITION)
+        return isAllowedToFile(FilingTypes.TRANSITION) && isAuthorized(AuthorizedActionsE.STAFF_FILINGS)
```

Comment:
I know very little about a Transition Filing. Is it staff-only? Do you know where the action button for it will be?

(If the action button will be under the Staff Filings menu then you will have to update isAuthorizedAddStaffFiling in dashboard.vue.)

Reply:
^^^ Only need to do this if we use a new authorized action for Transition Filing (I think).

Reply:
@Mihai-QuickSilverDev What do you know about Transition filings -- are they a staff-only filing? Where would the action button to start one be?